cn.feishu.Feishu.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: cn.feishu.Feishu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: feishu
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  # - --socket=wayland
+  - --socket=pulseaudio
+  # For ibus
+  # - --socket=session-bus
+  # - --socket=system-bus
+  # For camera
+  - --device=all
+  # For ptrace
+  - --allow=devel
+  - --filesystem=home
+separate-locales: false
+
+modules:
+  - name: feishu
+    buildsystem: simple
+    build-commands:
+      - install -D -m 755 -t /app/bin/ apply_extra feishu
+      - install -D -t /app/lib/ /usr/lib/x86_64-linux-gnu/{libbfd-*.so,libsframe.so.1}
+      - install -D -t /app/bin/ /usr/bin/ar
+      - install -D -t /app/share/icons/hicolor/scalable/apps/ cn.feishu.Feishu.svg
+      - install -D -t /app/share/applications/ cn.feishu.Feishu.desktop
+      - install -D -t /app/share/metainfo/ cn.feishu.Feishu.metainfo.xml
+    only-arches:
+      - "x86_64"
+    sources:
+      - type: script
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - source /app/extra/feishu/bytedance-feishu.sh
+          - exec zypak-wrapper /app/extra/feishu/feishu "$@"
+        dest-filename: feishu
+      - type: script
+        commands:
+          - ar x Feishu-linux.deb
+          - tar -xf data.tar.xz
+          - mv opt/bytedance/feishu .
+          - sed '/^exec -a/d' feishu/bytedance-feishu > feishu/bytedance-feishu.sh
+          - rm -rf opt usr *.tar.* debian-binary *.deb
+        dest-filename: apply_extra
+      - type: extra-data
+        filename: Feishu-linux.deb
+        url: https://sf3-cn.feishucdn.com/obj/ee-appcenter/6085d1c4/Feishu-linux_x64-6.9.20.deb
```

Comment:
is resdistribution EXPLICITLY allowed?

Reply:
No, so I have to install it via extra-data.